sources/packages/web/src/services/UserService.ts

```diff
@@ -37,4 +38,10 @@ export class UserService {
   async checkActiveUser(authHeader?: any): Promise<boolean> {
     return ApiClient.User.checkActiveUser(authHeader);
   }
+
+  async getInstitutionUser(
+    authHeader?: any,
```

Comment:
The auth header is being passed as a parameter in exceptional cases when the authentication token is not available globally yet. Just a few methods need this parameter. If this one does not need it, please consider removing it.

Reply:
The headers argument has been removed